corehq/form_processor/abstract_models.py

```diff
@@ -37,6 +37,10 @@ def set_submission_properties(self, submission_post):
     def to_json(self):
         raise NotImplementedError()
 
+    @property
+    def is_normal(self):
```

Comment:
one thing that sucks is that i can't add any attributes that are fields on the django model without it erroring (i.e. `is_error` etc.)